src/components/blogCard.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import styled from 'styled-components'
+import FeatureImage from 'images/default-feature-image.jpg'
+import ProfileIcon from 'images/DefaultProfileIcon.png'
+import { formatDate } from './helpers'
+
+const Card = styled.div`
+  margin: 15px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  min-width: 0;
+  word-wrap: break-word;
+  background-color: var(--white);
+  background-clip: border-box;
+  border-radius: 0.25rem;
+
+  border: 2px solid #bfbfbf;
+  border-radius: var(--border-radius);
+`
+
+const CardBody = styled.div`
+  flex: 1 1 auto;
+  min-height: 1px;
+  padding: 1.25rem;
+`
+
+const CardImgTop = styled.img`
+  width: 100%;
+  border-top-left-radius: calc(0.25rem - 1px);
+  border-top-right-radius: calc(0.25rem - 1px);
+`
+
+const Tag = styled.span`
+  color: var(--purple2);
+`
+
+const Small = styled.div`
+  font-size: 62.5%;
+`
+
+type BlogCardProps = {
+  title: String
+  reading_time: number
+  published_at: Date
+  excerpt: String
+  feature_image: String
+  slug: String
+  tags: Array<any>
+  primary_author_name: String
+  authors: Array<any>
```

Comment:
Can you also specify interfaces for Tags and Authors? (and any other interfaces that might be required).